### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by jpetrell**

### CODE REVIEW

File: pages/settings/devicelist/temperature/PageTemperatureSensor.qml [hunk 1]
```diff
@@ -82,7 +82,7 @@ Page {
 						id: batteryVoltageLabel
 						anchors.verticalCenter: parent.verticalCenter
 						font.pixelSize: Theme.font_size_body2
-						value: batteryVoltage.value
+						value: batteryVoltage.value == undefined ? NaN : batteryVoltage.value
```
Comment (jpetrell) [inline pages/settings/devicelist/temperature/PageTemperatureSensor.qml:85]: `Unable to assign [undefined] to double`

File: components/dialogs/InverterChargerModeDialog.qml [hunk 2]
```diff
@@ -44,7 +44,7 @@ ModalDialog {
 
 		RadioButtonControlValue {
 			enabled: modelData.enabled
-			visible: modelData.visible
+			visible: modelData.visible !== false
```
Comment (jpetrell) [inline components/dialogs/InverterChargerModeDialog.qml:47]: "Unable to assign [undefined] to bool"

File: pages/NotificationLayer.qml [hunk 3]
```diff
@@ -12,7 +12,7 @@ Item {
 	anchors.fill: parent
 
 	function showToastNotification(category, text, autoCloseInterval = 0) {
-		var toast = toaster.createObject(toastItemsModel, { "category": category, "text": text, autoCloseInterval: autoCloseInterval })
+		var toast = toaster.createObject(root, { "category": category, "text": text, autoCloseInterval: autoCloseInterval })
```
Comment (jpetrell) [inline pages/NotificationLayer.qml:15]: "Created graphical object was not placed in the graphics scene."

Tested that the toast is still shown and deleted like expected.

File: components/dialogs/CurrentLimitDialog.qml [hunk 4]
```diff
@@ -19,7 +19,7 @@ NumberSelectorDialog {
 		open()
 	}
 
-	title: Global.acInputs.currentLimitTypeToText(inputSettings ? inputSettings.inputType : 0)
+	title: inputSettings ? Global.acInputs.currentLimitTypeToText(inputSettings.inputType) : ""
```
Comment (jpetrell) [inline components/dialogs/CurrentLimitDialog.qml:22]: "Unrecognized AC Input Type"

**Review by blammit — APPROVED** [review]

**Review by chriadam — APPROVED** [review]
Good catches, thanks

**Pull request merged**